### PR TITLE
Add Initial Send Email Layout

### PR DESCRIPTION
**Pull request by jacknugent**
Desktop:
image
image

Mobile:
image

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "@types/react": "^17.0.18",
     "@types/react-dom": "^17.0.9",
     "bootstrap": "^5.1.0",
+    "markdown-to-jsx": "^7.1.3",
```
[the remark this answers is not in the document]
Comment (jacknugent) [reply frontend/package.json]: Per a discussion with @kevincianfarini, we're doing to convert the mock data to HTML, so I removed this now unnecessary package.

**Comment by kevincianfarini** [comment]
Can we made the prompts section an `<ul>` instead of using `*`?

**Review by kevincianfarini — APPROVED** [review]
lgtm, just one small nit

**Comment by jacknugent** [comment]
> Can we made the prompts section an `<ul>` instead of using `*`?

Good idea, done

**Pull request merged**